croud/util.py

```diff
@@ -91,3 +91,11 @@ def clean_dict(data: Dict) -> Dict:
         else:
             cleaned[k] = v
     return cleaned
+
+
+def confirm_prompt(msg):
+    msg = f"{msg.rstrip()} [yN] "
+    resp = input(msg).lower()
+    if resp in {"y", "yes"}:
```

Comment:
Cosmetics: the `in` already returns a boolean value, you can write it short as 

```suggestion
    return resp in {"y", "yes"}
```